### ISSUE TITLE
the code problem

### ISSUE THREAD

**Issue by NHZlX** [PureDiors/pytorch_RFCN#2]
I didn't run your code, but I have read the source code and found that 
`
self.rfcn_score = Conv2d(1024,7*7*8, 1,1, bn=False)
self.rfcn_bbox = Conv2d(1024, 7*7*self.n_classes,1,1,bn=False)`

Shouldn't it be

`
self.rfcn_score = Conv2d(1024,7*7*self.n_class, 1,1, bn=False)
self.rfcn_bbox = Conv2d(1024, 7*7*8,1,1,bn=False)`

?

**Comment by DuinoDu** [comment]
I agree with you, @NHZlX

**Comment by CosmosShadow** [comment]
This help me. Training on another dataset, this will works wrong.
Thank you.

**Comment by yangwf1** [comment]
why should it be self.rfcn_bbox = Conv2d(1024, 7*7*8,1,1,bn=False), and not Conv2d(1024, 7*7*4,1,1,bn=False) as clarified in the paper

**Comment by princevandev** [comment]
@yangwf1 in this implement, first 4 channels(0 to 3) are set for "background bbox prediction" even it is no used in training (see variable [rpn_bbox_inside_weights](https://github.com/PureDiors/pytorch_RFCN/blob/master/faster_rcnn/rfcn.py#L112) , it is to mask bg background boxes), because there is no boxes target for bg, it seems useless in bbox training. And following 4-7 channels are for fg boxes regression.

btw, My [R-FCN.pytorch](https://github.com/princewang1994/R-FCN.pytorch) has released for R-FCN and CoupleNet implement, welcome staring :)
